Dockerfile

```diff
@@ -31,7 +31,8 @@ RUN pip install -r requirements.txt
 
 COPY . /workspace
 
-RUN md5sum -c md5sums.txt # check file integrity
+# check file integrity
+RUN md5sum -c md5sums.txt
```

Comment:
Do you think it's a good idea to move md5sum checking above `COPY`? This would avoid redundant md5sum checking when a file changes in workspace, and is helpful for preventing accidental checking non-asset files in md5sum (assuming we only check downloaded asset files).

Reply:
Great idea!

Reply:
We discussed this offline. We need the copy to get the md5sums.txt file so we will leave this

Reply:
Yeah, that's right.